### PR TITLE
add cef console button to developer settings

### PR DESCRIPTION
**Pull request by Witherking25**

### CODE REVIEW

**Comment by Witherking25** [comment]
oop my ide did a bad job with the commit username lol

**Comment by PartyWumpus** [comment]
Works well, although it would probably be more convinient if it went directly to the `SharedJSContext` page, as no other pages would actually be useful as far as i can tell.
From the python side, a way of getting a link directly there would be `(await get_tab('SharedJSContext')).id` (which returns a value along the lines of `E88A4C68268596E8826CB5C08DA8F5FE`). 
Then in ts, run `NavigateToExternalWeb("localhost:8080/devtools/inspector.html?ws=localhost:8080/devtools/page/"+id)`
I did find tab.ws_url, but it returns `ws://localho...` not `ws=localho..`.

Okay I'm just going to try do this myself at this point. If it fails then I guess it can fall back to localhost:8080.

Clip of it working just to be sure:

https://user-images.githubusercontent.com/48649272/236269281-a96f2446-13f2-491e-9c5a-206fac6663cd.mp4

**Comment by Witherking25** [comment]
Oh neat, a pr to a pr? Lol

**Comment by PartyWumpus** [comment]
I guess so lol. Can't edit this directly so I'll make a PR.

**Comment by Witherking25** [comment]
yea having it directly open the console would be preferable

**Comment by Witherking25** [comment]
also need italian translations for the new ui elements, but probably let the guy that did the i18n stuff do that as i believe he made the ones for the rest of the ui if i remember correctly

**Comment by PartyWumpus** [comment]
I feel a little stupid right now, but your repo is protected so I can't directly edit it (unlike most PRs), and I can't make a fork of your fork without deleting my existing one (which I can't do). I guess I could make a fork under the steamdeckhomebrew team, but that seems silly.

**Comment by Witherking25** [comment]
i can give you write access if you want

**Comment by Witherking25** [comment]
wait would this fix?
Disable branch protection rules

**Comment by PartyWumpus** [comment]
Yeah that would probably work, sounds good.

**Comment by Witherking25** [comment]
ok try now

**Comment by Witherking25** [comment]
here i just added you to the repo so you can just push to it lol

**Comment by Witherking25** [comment]
nice

**Comment by PartyWumpus** [comment]
Hopefully someone else has the time to test this, because I don't know if I can test my own code lol.
I'll still give it a test once it's finished building though.

**Comment by Witherking25** [comment]
works great for me

**Comment by PartyWumpus** [comment]
did you just accidentally overwrite your fork?

**Comment by PartyWumpus** [comment]
It's working well on my PC and my deck

https://user-images.githubusercontent.com/48649272/236298266-95ff1e63-8250-4ef5-94af-0626258a3fc4.mp4

**Review by JohnnyCrazy — CHANGES_REQUESTED** [review]
Just 2 small things, otherwise LGTM

File: frontend/src/components/settings/pages/developer/index.tsx
Comment (Witherking25) [reply frontend/src/components/settings/pages/developer/index.tsx:87]: oops

File: frontend/src/components/settings/pages/developer/index.tsx
Comment (Witherking25) [reply frontend/src/components/settings/pages/developer/index.tsx:85]: i like being verbose with my parentheses, but i can remove them if you like :)

**Review by TrainDoctor — APPROVED** [review]
Tested on my deck, works good. Once Johnny approves this will be good for merge.

**Review by JohnnyCrazy — APPROVED** [review]

**Pull request merged**
